### DISCUSSION TITLE
Open a program with "Service not implemented: CreateDesktop"

### DISCUSSION THREAD

**Discussion by linguaccia22** [sandboxie-plus/Sandboxie#2020] [Documents and notes]
Hi everyone I use Sandboxie Plus v1.2
I was trying to start a program in the virtual area but the program is not working properly does not let me log in.
This error comes out:

poker.exe (15164): SBIE2205 Service not implemented: CreateDesktop

How can I solve? a thousand thanks

**Comment by isaak654** [comment]
Try adding the following line below your [sandbox] section in Sandboxie.ini file: https://github.com/sandboxie-plus/Sandboxie/issues/1442#issuecomment-1170091941

**Reply by linguaccia22** [reply]
Thanks a lot ^^
